docs/mobile-apps/features/instant-booted-simulators.md

```diff
@@ -0,0 +1,391 @@
+:::
+
+| Device Name              | Platform Version                                |              Result              |   Instant Booted   | Note                                                                                                                                                                         |
+| :----------------------- | :---------------------------------------------- | :------------------------------: | :----------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| iPhone Instant Simulator | latest/current_major                            |       iPhone 14 with 16.2        | :white_check_mark: | The requested instant-booted device alias, requested OS version, and latest OS version installed in our cloud match. This means an instant-booted Simulator will be started. |
+| iPad Instant Simulator   | latest/current_major                            | iPad (10th Generation) with 16.2 | :white_check_mark: |                                                                                                                                                                              |
+| iPhone Instant Simulator | previous_major                                  |       iPhone 13 with 16.2        | :white_check_mark: |                                                                                                                                                                              |
+| iPad Instant Simulator   | previous_major                                  | iPad (9th Generation) with 15.5  | :white_check_mark: |                                                                                                                                                                              |
+| iPhone Instant Simulator | A fixed iOS version, for example, 16.2, or 15.5 |                -                 |        :x:         | The combination is invalid. The device name alias **CAN ONLY** be combined with platform version aliases                                                                     |
+| iPad Instant Simulator   | A fixed iOS version, for example, 16.2, or 15.5 |                -                 |        :x:         |                                                                                                                                                                              |
+
+### Using explicit capabilities
+
+You can also use explicit capabilities to access the Instant Booted iOS Simulators. This is not recommended as it is more likely to land on a normal booted Simulator when Apple releases a new minor or major version of iOS. This means that you will not automatically get access to the latest device and or iOS version for your tests and will be testing your app on what is most likely not the most popular device and or iOS version used by your users.
```

Comment:
```suggestion
You can use explicit capabilities to access the Instant Booted iOS Simulators. However, this is not recommended because you're more likely to end up on a normally booted simulator when Apple releases a new minor or major version of iOS. As a result, you won't automatically gain access to the latest device or iOS version for your tests, and you'll likely be testing your app on a less popular device or iOS version that is not commonly used by your users.
```